csharp/src/Apache.Arrow.Flight/Generated/Flight.cs

```diff
@@ -0,0 +1,3536 @@
+// <auto-generated>
```

Comment:
Why are you checking generated code in? You should be able to reference Grpc.Tools and generate this file automatically from the proto file.

Reply:
I had problems in the build process, with sourcelink, I wrote it in the description, sourcelink could not find the auto generated code.

If you know a fix I would greatly appreciate it :)

Reply:
I haven't heard of this problem before.

Could you provide some more detail about how didn't it work?

Reply:
here is the build where I saw the errors: https://github.com/apache/arrow/runs/1412215805

Error log extract:
```
2 Documents with errors:
2d1a642b553a64701a2cd2463307ee00b117363d81a8914a91a6e34b6bae0700 sha256 csharp /Users/runner/work/arrow/arrow/csharp/src/Apache.Arrow/obj/Debug/netcoreapp2.1/Format/Flight.cs
https://raw.githubusercontent.com/apache/arrow/afa5d50e94e7f75d8973aa8022502f47b93740ef/csharp/src/Apache.Arrow/obj/Debug/netcoreapp2.1/Format/Flight.cs
error: url failed NotFound: Not Found
36ececbfdcef40763e3f3effdc3fc42392fd80a6fad48bf431395d0e705d7950 sha256 csharp /Users/runner/work/arrow/arrow/csharp/src/Apache.Arrow/obj/Debug/netcoreapp2.1/Format/FlightGrpc.cs
https://raw.githubusercontent.com/apache/arrow/afa5d50e94e7f75d8973aa8022502f47b93740ef/csharp/src/Apache.Arrow/obj/Debug/netcoreapp2.1/Format/FlightGrpc.cs
error: url failed NotFound: Not Found
sourcelink test failed
```

I have not used sourcelink much myself, but it seems like it is linking it to a github file in this case, so I guessed that it is required to be checked in for it to work.

That atleast solved the issue of building (but I know that it is a very ugly fix/workaround), I searched a bit if I can change the build process is any way. But could not find anything yet.

Reply:
I think `<EmbedUntrackedSources>true</EmbedUntrackedSources>` will fix this.

Reply:
Ok, I will give it a shot directly, thanks!

Reply:
That did the trick, thank you! I removed the generated files and added Grpc.Tools instead.